01_fib/choohy/main_test.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestFibNeg11(t *testing.T) {
```

Comment:
Good work adding a few more test cases. One thing I would suggest though, is that these can be combined into a single Test function. You can see that the structure of each of the 3 tests below is the exact same, with the only difference being the input to the function, and the expected output. This is a good sign that some abstraction can be used to refactor these together and clean up your code. 

The way to do this would be to create a `map` of cases which you want to test against, and then range over those to achieve this.

Check out [Multiple Test Cases](https://goplay.space/#Zy2ta-z8vrO) for some inspiration and see if you can achieve something similar.